source/includes/_security.md

```diff
@@ -447,13 +533,25 @@ $body->setRoles(['TestUpdatedRole']);
 
 $apiInstance->updateUserRoles($body,$xKillbillCreatedBy,$xKillbillReason,$xKillbillComment);
 ```
-> Example Response:
 
-```json
+**Query Parameters**
+
+None
+
+**Response**
+
+If successful, returns a status code of 204 and an empty body.
 
```

Comment:
A **Request Body** section would be required I think. Can you please confirm and add this if required?